tests/unit/run_unit_tests.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
+set -e
+
 coverage run --source=oioswift -a $(which nosetests) -v tests/unit/controllers
 coverage run --source=oioswift -a $(which nosetests) -v \
```

Comment:
We should run only one big batch of tests, or the second batch will not run if the first fails.